lib/sgtn-client/api/translation.rb

```diff
@@ -80,6 +81,7 @@ def self.getStrings(component, locale)
       private
 
       def self.getTranslation(component, key, locale)
+        locale = SgtnClient::LocaleUtil.get_best_locale(locale)
         items = get_cs(component, locale)
```

Comment:
```suggestion
        items = getStrings(component, locale)
```

Reply:
Looks both work. This PR is to fix bug, let's not change this if there isn't a bug.